plugins/scaffolder-node/src/actions/executeShellCommand.ts

```diff
@@ -29,8 +30,10 @@ export type ExecuteShellCommandOptions = {
   args: string[];
   /** options to pass to spawn */
   options?: SpawnOptionsWithoutStdio;
-  /** stream to capture stdout and stderr output */
-  logStream?: Writable;
+  /** logger to capture stdout and stderr output */
+  logger?: Logger;
+  /** log level to use for logging output */
+  logLevel?: LeveledLogMethod;
```

Comment:
Not sure about introducing this, what would be better would be that if people want to change the `logLevel` they can provide the deprecated `logStream`, and we should fix the levels that are emitted to logger.